cmake/add_enclave_test.cmake

```diff
@@ -4,9 +4,7 @@
 
 ## This function is to add test for given host file and enclave file.
 ## TEST_NAME	: test name for add test.
-## HOST_SUBPATH : Path of host file directory.
 ## HOST_FILE	: Host application executable file name.
-## ENC_SUBPATH	: Path of enclave file directory.
 ## ENC_FILE		: Signed/UnSigned enclave file name.
 ## DESCRIPTION : For ADD_WINDOWS_ENCLAVE_TESTS enabled function will copy signed 
```

Comment:
Is description an argument to `add_enclave_test`? I don't think so, but this is confusing. Would you rewrite this please? 😄